endaq/calc/stats.py

```diff
@@ -19,6 +19,8 @@ def shock_vibe_metrics(
         df: pd.DataFrame,
         tukey_percent: float = 0.1,
         highpass_cutoff: float = None,
+        accel_units: str = "gravity",
+        disp_units: str = "in",
         freq_splits: np.array = [0, 65, 300, 1500, None],
```

Comment:
There are a couple of problems with this; `np.array` isn't a type (it's a function that returns one of several array types, depending on content) and a list as a default keyword argument is recommended against, since it is mutable (meaning it could get changed accidentally).

Suggestion: `freq_splits: typing.Union[np.ndarray, list, tuple] = (0, 65, 300, 1500, None),`
Then, at the start of the function, do `freq_splits = np.array(freq_splits)` (it will work whichever data type is used).

Reply:
thanks, I was wondering about how to define a list!